apps/src/templates/projects/PersonalRecentProjects.jsx

```diff
@@ -0,0 +1,70 @@
+import React, {PropTypes} from 'react';
+import ProjectCard from './ProjectCard.jsx';
+import {personalProjectDataPropType} from './projectConstants';
+import color from "../../util/color";
+import i18n from "@cdo/locale";
+import Radium from 'radium';
+
+const styles = {
+  grid: {
+    width: '100%'
+  },
+  link: {
```

Comment:
Should we use the styling for the linkToViewAll in ContentContatiner here, so it's consistent?